language_service/src/protocol.rs

```diff
@@ -2,15 +2,18 @@
 // Licensed under the MIT License.
 
 use qsc::line_column::Range;
-use qsc::{compile::Error, target::Profile, LanguageFeatures, PackageType};
+use qsc::PackageType;
+use qsc::{compile::Error, target::Profile, LanguageFeatures};
+use qsc_linter::LintConfig;
```

Comment:
Would it make sense to re-export this via a `pub use` in qsc to keep with the pattern of everything outside of the compiler accessing things only via that crate? I know we haven't been vigilant about keeping that up (case in point, `qsc_project` is directly referenced below, but maybe that is a special case), so feel free to defer, but just thought I'd mention it.

Reply:
Great! Yeah, things have to be public from the underlying qsc_* crates to be visible in qsc, so nothing stops something outside of the compiler folder from importing from there, nor could we reasonably enforce such a limit with anything other than code review and periodic refactoring. Sounds like you already have the types you need for this spot, so no other changes should be needed.